src/js/common/schemaform/save-load-actions.js

```diff
@@ -137,11 +137,11 @@ export function migrateFormData(savedData, savedVersion, migrations) {
  * Saves the form data to the back end
  * @param  {String}  saveType  The type of save that's happening, auto or save and redirect
  * @param  {String}  formId    The form’s formId
+ * @param  {Object}  formData  The data the user has entered so far
  * @param  {Ingeter} version   The form’s version
  * @param  {String}  returnUrl The last URL the user was at before saving
- * @param  {Object}  formData  The data the user has entered so far
  */
-function saveForm(saveType, formId, version, returnUrl, formData) {
```

Comment:
This should match the order (minus saveType) in these places:

https://github.com/department-of-veterans-affairs/vets-website/blob/ab08c2beb37c1e0d744bd28ca2cf1934fdd6941e/src/js/common/schemaform/SaveFormLink.jsx#L56
https://github.com/department-of-veterans-affairs/vets-website/blob/ab08c2beb37c1e0d744bd28ca2cf1934fdd6941e/src/js/common/schemaform/FormPage.jsx#L120
https://github.com/department-of-veterans-affairs/vets-website/blob/ab08c2beb37c1e0d744bd28ca2cf1934fdd6941e/src/js/common/schemaform/review/ReviewPage.jsx#L106